src/pages/docs/deployments/kubernetes/automatically-track-third-party-helm-charts/index.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-05-02
+modDate: 2024-05-02
+title: Automatically tracking third party helm charts
+description: External feed triggers allow you to automatically create a new release as a result of new container images or helm charts being pushed to their respective repositories.
+navOrder: 41
+---
+
+With a growing number of applications being provided with Helm charts as a primary method of installation, often all that needs to be done is a `helm install` against your cluster and the application will be up and running.
+
+However managing updates can be a more involved process. Not only do you need to know when a new release is available, but you also need to have someone with credentials to run the `helm upgrade` against your cluster. Meaning you'll either need to share important credentials among everyone performing updates or have only a few people busy performing these updates.
+
+Octopus Deploy provides a full workflow to manage updates, either hands on or fully hands off.
+
+### Setting up the project
+
+A Helm chart deployment like this is simple with Octopus Deploy.
+
+1. Start with the **Deploy a Helm chart** step
+2. Link it to the required Kubernetes clusters via target tags
```

Comment:
It might be good to add a link to the target tag page, given that they've recently been renamed from roles.

Reply:
Good idea